recipes/linkchecker/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "linkchecker" %}
+{% set version = "10.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/LinkChecker-{{ version }}.tar.gz
+  sha256: 943a3f3f30279fbb85fc6f0569d808cf1ae4be0de4c2050c6f713429b8e58b59
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - linkchecker = linkcheck.command.linkchecker:linkchecker
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    # Linkchecker uses hatchling and the hatch-vcs plugin to build
+    - hatchling
+    - hatch-vcs
```

Comment:
Even though there is this build setup, linkchecker is a pure python package without any compiled bits.